img/warning.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can we just use the existing img/exclamation-mark.svg imagery?

Reply:
yep, i didn't even realize that was there!

i'll check it in a bit today